src/frontends/onnx/frontend/src/op/layer_normalization.cpp

```diff
@@ -43,58 +32,42 @@ ov::OutputVector layer_normalization(const ov::frontend::onnx::Node& node) {
                      num_inputs == 2 || num_inputs == 3,
                      "LayerNormalization expects 2 or 3 input tensors. Got: ",
                      num_inputs);
+    CHECK_VALID_NODE(node,
+                     node.get_outputs_size() == 1,
+                     "LayerNormalization expects 1 output tensor to be used in a model, other configurations are used "
+                     "for training and are not supported. Got: ",
+                     node.get_outputs_size(),
+                     " outputs.");
 
-    const auto& X = inputs.at(0);
-    const auto& Scale = inputs.at(1);
-
-    auto axis = node.get_attribute_value<std::int64_t>("axis", -1);
-    double epsilon = node.get_attribute_value<double>("epsilon", 1e-5);
-    int64_t stash_type_i =
-        node.get_attribute_value<int64_t>("stash_type",
-                                          static_cast<int64_t>(TensorProto_DataType::TensorProto_DataType_FLOAT));
+    auto default_stash_type_i = static_cast<int64_t>(TensorProto_DataType::TensorProto_DataType_FLOAT);
+    int64_t stash_type_i = node.get_attribute_value<int64_t>("stash_type", default_stash_type_i);
     element::Type stash_type = common::get_ov_element_type(stash_type_i);
+    CHECK_VALID_NODE(node,
+                     stash_type == element::f32 || stash_type == element::bf16,
+                     "LayerNormalization `stash_type` attribute is expected to represent float or bfloat16 type. Got: ",
+                     stash_type);
```

Comment:
Why do we need this extra validation? Just to do ONNX job? We don't really care about the type, our conversion code will be the same. Next time when LayerNormalization ONNX operator's specification is update, we need to change this code on our side.

Reply:
Updated code. Please review